addons/rose/addon/styles/rose/variables/color/_css.scss

```diff
@@ -123,9 +123,9 @@
   --ui-border-subtler-1: #{color(ui-gray, 800)};
   --ui-border: #{color(ui-gray, 700)};
 
-  --ui-gray-subtler-6: #{color(ui-gray, 900)};
-  --ui-gray-subtler-5: #{color(ui-gray, 800)};
-  --ui-gray-subtler-4: #{color(ui-gray, 700)};
+  --ui-gray-subtler-6: #{color(ui-gray, 950)};
+  --ui-gray-subtler-5: #{color(ui-gray, 900)};
+  --ui-gray-subtler-4: #{color(ui-gray, 800)};
```

Comment:
I only made these ones darker in dark mode, I couldn't find good examples of the lighter ones being used so left them as is. Let me know if you think I should also adjust them.